.chloggen/pkg-stanza-missing-storage-panic.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: pkg/stanza, filelog, journald, windowseventlog
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Fix issue where specifying a non-existent storage extension caused panic during shutdown.
```

Comment:
This change appears unrelated. Does this need to be rebased?

Reply:
Thanks, I've rebased to exclude this